### PR TITLE
Major field to job model

### PR DESCRIPTION
**Pull request by eldaraharonson**
1. Added 'Major' field to 'Job' model.
2. Updated example data in 0002 migration file to include 'Major' field.
3. Added a function to Job model that returns all jobs corresponding to a specific major.
4. Added a test for that function.
5. Put nohup.out in .gitignore

### CODE REVIEW

**Review by guyy13 — CHANGES_REQUESTED** [review]
No need to push the nohup.out file

**Comment by guyy13** [comment]
> No need to push the nohup.out file

Besides that, looks good

File: jobboard/models.py [hunk 1]
```diff
@@ -109,5 +111,13 @@ def get_jobs_by_recruiter_id(cls, recruiter: Recruiter):
         """
         return set(cls.objects.filter(recruiter__user_id=recruiter.user.id))
 
+    @classmethod
+    def get_jobs_by_major(cls, major):
```
Comment (yaelerdr) [inline jobboard/models.py]: I think we might need to define the function as 
`def get_jobs_by_major(cls, *major)`
so the student will be able to filter jobs according to more than 1 major, and of course, change the functionality also, properly

**Review by yaelerdr — CHANGES_REQUESTED** [review]

File: jobboard/models.py (hunk 1 above)
Comment (eldaraharonson) [reply jobboard/models.py]: On it

**Comment by eldaraharonson** [comment]
> No need to push the nohup.out file

Fixed it

File: jobboard/models.py [hunk 2]
```diff
@@ -109,5 +111,16 @@ def get_jobs_by_recruiter_id(cls, recruiter: Recruiter):
...
 
+    @classmethod
+    def get_jobs_by_major(cls, *majors):
+        """
+        Allows students to search for a job based on one or more university majors
+        majors parameter holds Major instances
+        """
+        jobs_to_return = set()
+        for major in majors:
+            jobs_to_return.update(set(cls.objects.filter(major=major)))
```
Comment (guyy13) [inline jobboard/models.py]: Instead of the for loop you can do this in one line like so:
`cls.objects.filter(major__in= majors)`

**Review by guyy13 — CHANGES_REQUESTED** [review]

**Review by YarinBou — COMMENTED** [review]
great job eldar

**Review by benmenachem — APPROVED** [review]

**Comment by guy9050** [comment]
@eldaraharonson there are some unresolved comments ^

File: jobboard/models.py (hunk 2 above)
Comment (eldaraharonson) [reply jobboard/models.py]: Fixed it

**Review by guyy13 — APPROVED** [review]
nice work!

**Review by YarinBou — APPROVED** [review]

**Review by yaelerdr — APPROVED** [review]
looks good :)

**Comment by eldaraharonson** [comment]
> @eldaraharonson there are some unresolved comments ^

They are resolved now.

**Comment by guy9050** [comment]
please squash the commits

**Comment by eldaraharonson** [comment]
> please squash the commits

Done!

**Pull request merged**